public/api/conf/1.0/schemas/state-pension.json

```diff
@@ -230,6 +230,11 @@
       "type": "boolean",
       "description": "Whether the customer has a \u2018reduced rate election\u2019 in force at the beginning of the tax year 35 years before the tax year of State Pension age",
       "example": "true"
+    },
+    "abroadAutoCredits": {
+      "type": "boolean",
+      "description": "Whether the customer is male, reached State Pension age before 6 October 2018 and has an overseas address",
```

Comment:
This does not explain the context of the of the flag and what it means for the consumer.

Reply:
This should explain to the API consumer what to do (exclude them) and why they have to do so.